src/Tizen.NUI.Scene3D/src/internal/CameraBindableProperty.cs

```diff
@@ -0,0 +1,241 @@
+/*
+ * Copyright(c) 2022 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+using System.ComponentModel;
+using Tizen.NUI.Binding;
+
+namespace Tizen.NUI.Scene3D
+{
+    public partial class Camera
+    {
+        /// <summary>
+        /// TypeProperty
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        internal static readonly BindableProperty TypeProperty = BindableProperty.Create(nameof(Type), typeof(CameraType), typeof(Tizen.NUI.Scene3D.Camera), CameraType.FreeLook, propertyChanged: (bindable, oldValue, newValue) =>
+        {
+            var instance = (Tizen.NUI.Scene3D.Camera)bindable;
+            if (newValue != null)
+            {
+                instance.InternalType = (CameraType)newValue;
+            }
+        },
+        defaultValueCreator: (bindable) =>
+        {
+            var instance = (Tizen.NUI.Scene3D.Camera)bindable;
+            return instance.InternalType;
+        });
+
+        /// <summary>
+        /// ProjectionModeProperty
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        internal static readonly BindableProperty ProjectionModeProperty = BindableProperty.Create(nameof(ProjectionMode), typeof(ProjectionMode), typeof(Tizen.NUI.Scene3D.Camera), ProjectionMode.PerspectiveProjection, propertyChanged: (bindable, oldValue, newValue) =>
```

Comment:
could you remove `[EditorBrowsable(EditorBrowsableState.Never)]` for all `internal` APIs?
I can see many of these cases below.
thank you.

Reply:
Thank you I fixed it all